include/gz/sim/Server.hh

```diff
@@ -225,14 +225,42 @@ namespace gz
                   const SystemPluginPtr &_system,
                   const unsigned int _worldIndex = 0);
 
+      /// \brief Add a System to the server. The server must not be running when
+      /// calling this.
+      /// \param[in] _system system to be added
+      /// \param[in] _entity Entity of system to be added.
+      /// \param[in] _sdf Pointer to the SDF of the entity.
```

Comment:
Can you update the documentation of `_sdf` to state that it's the `sdf::Element` of a `<plugin>` tag with configuration options for the system being added?